client/src/components/NavBar.js

```diff
@@ -38,17 +39,34 @@ class NavBar extends Component {
       logout,
       handleSidebarHide
     } = this;
+    const landingPage = window.location.pathname === "/";
+
     return (
       <React.Fragment>
         <Menu size="mini" flued inverted>
-          <Menu.Item onClick={handleShowClick}>
-            <Icon size="large" name="list layout"></Icon>
-          </Menu.Item>
+          {landingPage ? (
+            <Icon
+              name="align left"
+              style={{
+                fontSize: "40px",
+                position: "absolute",
+                zIndex: " 3",
+                top: "25px",
+                color: "#2699fb"
+              }}
+              onClick={this.handleShowClick}
+            />
+          ) : (
+            <Menu.Item onClick={handleShowClick}>
+              <Icon size="large" name="list layout"></Icon>
+            </Menu.Item>
+          )}
```

Comment:
You need to remove the condition from here and use this Menu only ⬇️ because we handle the second Menu on LandingPage component.
```
           <Menu.Item onClick={handleShowClick}>
              <Icon size="large" name="list layout"></Icon>
            </Menu.Item>
```

Reply:
ok I have done